app.js

```diff
@@ -1,76 +1,107 @@
 #!/usr/bin/env node
 
-const { CLIEngine } = require('eslint');
+const { ESLint } = require('eslint');
 const path = require('path');
 const yargs = require('yargs');
 const write = require('write');
 const debug = require('debug')('eslint-output');
 const rc = require('./config');
 
 const cwd = path.resolve(process.cwd());
-const { maxWarnings, quiet } = yargs.options({
-  maxWarnings: { type: 'number', default: false },
-  quiet: { type: 'boolean', default: false },
+const { formatOverrides, maxWarnings, quiet, _ } = yargs.options({
```

Comment:
On first read, the underscore here was a little confusing as my mind went to lodash.

reading the docs i see this:

> Get the arguments as a plain old object.
> Arguments without a corresponding flag show up in the argv._ array. Note that elements of argv._ may be converted to numbers by default.

Can we give this a more contextual name so its easier to read?

maybe?

```suggestion
const { formatOverrides, maxWarnings, quiet, _: additionalArguments } = yargs.options({
```

Reply:
Yep, makes sense - added.